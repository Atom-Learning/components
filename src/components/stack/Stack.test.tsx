import { render } from '@testing-library/react'
import * as React from 'react'

import { Stack } from '.'

describe('Stack component', () => {
  it('renders', async () => {
    const { container } = render(<Stack />)

    expect(container).toMatchSnapshot()
  })
})
