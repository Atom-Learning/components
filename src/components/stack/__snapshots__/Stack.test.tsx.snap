// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stack component renders 1`] = `
@media  {
  .c-jgdwfn {
    display: flex;
  }

  .c-jgdwfn > * {
    margin: 0;
  }
}

@media  {
  .c-jgdwfn-ejCoEP-direction-row {
    flex-direction: row;
  }

  .c-jgdwfn-XefLA-wrap-true {
    flex-wrap: wrap;
  }

  .c-jgdwfn-awKDG-justify-start {
    justify-content: flex-start;
  }

  .c-jgdwfn-jroWjL-align-center {
    align-items: center;
  }
}

<div>
  <div
    class="c-jgdwfn c-jgdwfn-ejCoEP-direction-row c-jgdwfn-XefLA-wrap-true c-jgdwfn-awKDG-justify-start c-jgdwfn-jroWjL-align-center"
  />
</div>
`;
